src/_P040_ID12.ino

```diff
@@ -56,6 +55,19 @@ boolean Plugin_040(byte function, struct EventStruct *event, String& string)
         break;
       }
 
+    case PLUGIN_TIMER_IN:
+      {
+        if (Plugin_040_init) {
+            // Reset card id on timeout
+            UserVar[event->BaseVarIndex] = 0;
+            UserVar[event->BaseVarIndex + 1] = 0;
+            String log = F("RFID : Removed Tag");
+            addLog(LOG_LEVEL_INFO, log);
```

Comment:
Why send a log about "removed tag" during init?
And Why create a String object for it, as you don't modify the flashstring?
Just directly use it as parameter in addLog.  (which is a macro to also check the loglevel active)

Reply:
completely opposite here - we use it only when plugin is initiated, maybe this guard is not needed (timer is armed in already checked condition) but used it for consistent

Reply:
I was more surprised by the log to state "removed tag" when you simply initialize the task.

Reply:
corrected: plugin is already initiated